lib/environment_light.rb

```diff
@@ -11,5 +11,13 @@
 require 'bundler/setup'
 
 # Load application settings
-require File.expand_path("../../config/initializers/rails_config", __FILE__)
-Settings = RailsConfig.load_files ["settings.yml","settings.local.yml", "settings/#{Rails.env}.yml"].map{|f| "#{Rails.root}/config/#{f}" }
+require Rails.root.join('config/initializers/rails_config.rb')
+Settings = RailsConfig.load_files([
+    'settings.yml',
+    "settings/#{Rails.env}.yml",
+    "environments/#{Rails.env}.yml",
+    'settings.local.yml',
+    "settings.#{Rails.env}.local.yml",
+    "environments/#{Rails.env}.local.yml",
+    'hets.yml'
+  ].map { |f| Rails.root.join('config', f) })
```

Comment:
:-1:
This is "Repeat yourself" and the root cause inconsistencies: One should use the internally defined set of files (whatever this is) and prepend or append the other stuff. I already mentioned, why append is wrong above. BTW: Wondering, why hets.yml now gets always included? IIRC , e.g. the git stuff doesn't need it at all ...